spec/models/plugins/fundraiser_spec.rb

```diff
@@ -31,4 +31,15 @@
     expect( fundraiser.liquid_data[:donation_bands]).to eq "null"
   end
 
+  it 'serializes a named donation band' do
+    allow(PaymentProcessor::Currency).to receive(:convert)
+    _ = create :donation_band # We create this because it would be the default if the named band wasn't found.
```

Comment:
Yeah, I know, but when I'm doing something non-standard like this `_ =` is a signal to me at least that I'm aware that I'm doing something weird, but that weirdness is serving a purpose. Maybe the explicitly unused assignment and the comment is overkill. :)